pkg/virt-handler/vm.go

```diff
@@ -508,7 +508,7 @@ func (d *VirtualMachineController) hasGracePeriodExpired(dom *api.Domain) (hasEx
 	hasExpired = false
 	timeLeft = 0
 
-	if dom == nil {
+	if dom == nil /* || dom.Spec.Metadata.KubeVirt.GracePeriod == nil*/ {
```

Comment:
@xpivarc   
now       "should delete running Domains if no cluster wide equivalent exists and no grace period info exists".  unit test case  is failing .
i am thinking add pkg/virt-handler/vm.go +2428 , call hasGracePeriodExpired  if  GracePeriod != nil
what do you think , is it correct place to add that ?

Reply:
The test seems to pass

Reply:
ya ..

Reply:
could you give  approve/lgtm   this PR .